### PR TITLE
Return optimizer without learning scheduler

### PR DESCRIPTION
**Pull request by Riccardinho22**

### CODE REVIEW

File: src/pl_modules/model.py [hunk 1]
```diff
@@ -73,6 +73,8 @@ def configure_optimizers(
         opt = hydra.utils.instantiate(
             self.hparams.optim.optimizer, params=self.parameters(), _convert_="partial"
         )
+        if self.hparams.optim.use_lr_scheduler is False:
```
Comment (lucmos) [inline src/pl_modules/model.py:76]: Thanks for the PR!

A minor comment, the `is` operator checks whether two variables point to the same object in memory.
Thus, I think this should be `self.hparams.optim.use_lr_scheduler is False` or even better 
```python
if not self.hparams.optim.use_lr_scheduler:
```

Comment (Riccardinho22) [reply src/pl_modules/model.py:76]: Ok thanks for the suggestion :)
